### ISSUE TITLE
How to train multiple categories?

### ISSUE THREAD

**Issue by xncaffe** [liuwei16/CSP#23]
Hi，Liu Wei. I have mastered the use of CSP. But I don't seem to find a way to train it to train multiple categories of data sets (such as VOC). can you help me? @liuwei16

**Comment by lwpyr** [comment]
I think it is not a good idea to use this box free strategy in complex object detection. This scale regression strategy works in simple object detection tasks like pedestrians and faces.
If you insist on training this on VOC, you need to change the class prediction heat map's channel from 1 to N_class. Also you can specify the height and width regression for each category. And softmax cross entropy will replace the BCEloss in loss function.
